frontend/lib/build_config/build_config.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO: Currently unused
```

Comment:
Can you maybe add a comment what this file is for and what it does (and what it does not)?